src/cascadia/TerminalSettingsModel/TerminalSettings.cpp

```diff
@@ -67,10 +67,18 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         THROW_HR_IF_NULL(E_INVALIDARG, profile);
 
         const auto globals = appSettings.GlobalSettings();
-        settings->_ApplyProfileSettings(profile, globals.ColorSchemes());
+        settings->_ApplyProfileSettings(profile);
         settings->_ApplyGlobalSettings(globals);
+        settings->_ApplyAppearanceSettings(profile.DefaultAppearance(), globals.ColorSchemes());
 
-        return *settings;
+        if (profile.UnfocusedAppearance())
+        {
+            auto child = settings->CreateChild();
+            child->_ApplyAppearanceSettings(profile.UnfocusedAppearance(), globals.ColorSchemes());
+            return winrt::make<TerminalSettingsStruct>(*settings, *child);
+        }
+
+        return winrt::make<TerminalSettingsStruct>(*settings, nullptr);

```

Comment:
We want to be able to check (in the TermControl side) whether an `UnfocusedSettings` even exists, otherwise we will go through the entire re-rendering process simply to update the appearance to what it already is